rosetta-source/src/main/rosetta/base-staticdata-asset-common-type.rosetta

```diff
@@ -95,13 +158,43 @@ type IndexReferenceInformation: <"A class defining information related to Index"
     condition IndexAttributes: <"A required choice condition for either a floating rate or inflation rate index.">
         indexName exists or indexId exists
 
-type Index extends ProductBase: <"Identifies an index by referencing a product identifier.">
+// ART-02 Observable and Index refactoring
```

Comment:
**PARTIALLY AGREE**

- AGREE with attributes listed below for new type Observable
- REMARK current list may need to be extended with missing one that is Fixed Rate 

**PROPOSAL**
- add one addionalt attribute for Observable that would be of type FixedRate
- **RATIONALE** : cumulative reason below :
  - considering that TradeLot->PriceQuantity->Observable is the generic place where "arguments" of Payout shall be defined (with metadata location) for being referenced in Payout (with metadata address), then for any kinds of payout argument all would be an Observable, except the Fixed Rate, that is to say accross the whole CDM model FixedRatePayout would be the unique Payout component which argument would not be an Observable (by "argument" please hear "X" in "F(X)" when "F" is the functional formula of a given Payout)
  - despite the fact it is "already observed" (because it is given as Fixed value), a Fixed Rate is all in all homegneous _in nature_ to floatingRateIndex, that is to say it is same class "Rate", that is to say fact a Rate is already Fixed or not Fixed yet ("Floating") does not make a difference from a payout "F(X") point of view

Reply:
Makes sense.

Do you see "fixed rate" as a new attribute of `Observable` or another type of `Index`?

That is, either:

`type Observable: `
`    asset Asset (0..1)  `
`   quotedCurrencyPair QuotedCurrencyPair (0..1)`
`   index Index (0..1)`
`   fixedRate number (0..1)`
`      condition: one-of `

or 

`type Index: <"Identifies an index by referencing a product identifier.">`
`    fixedRate number (0..1)  `  
`    floatingIndex FloatingIndex (0..1) `
`    inflationIndex InflationIndex (0..1) `
`   equityIndex EquityIndex (0..1)`
`    creditIndex CreditIndex (0..1) `
`     condition: one-of`

Reply:
would recommand to define it as a new **attribute of Observable**, because there is no issuer and no other features, per se that is only what it is : a fixed rate... 

yet would also avoid usage of type _number_ 
- that is a "poor" type if i may say, compared to PriceSchedule type where datedValue might be defined if need be, etc. 
- also let's consider that type used in current CDM for fixed rate (when defining a fixed rate via TradeLot->PriceQuantity to be referenced then in Payout leg) is PriceSchedule type, so using poor type such as number instead would deprecate such original type for no good reason in my view...

that being said, proposal would be as below :

type **Observable**: 
   asset Asset (0..1) 
   quotedCurrencyPair QuotedCurrencyPair (0..1)
   index Index (0..1)
   fixedRate **PriceSchedule** (0..1)
     condition: one-of